ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1076,18 +1076,24 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
                           accessoryType: TableViewCellAccessoryType = .none) {
         if let attributedTitle = attributedTitle {
             titleLabel.attributedText = attributedTitle
+            isUsingAttributedTitle = true
         } else {
             titleLabel.text = title
+            isUsingAttributedTitle = false
```

Comment:
For clarification: the additional reaffirmation of assigning the `isUsingAttributed` text variables `false` is necessary because the cells seem to keep the old values and makes it show into the other cells. This change is essentially a double-check safety.